### DISCUSSION TITLE
Should Calibrator pre-built Settings be lower case?

### DISCUSSION THREAD

**Discussion by andreamari** [unitaryfund/mitiq#2015] [General]
When using use the `Calibrator` one typically must import some pre-made instances of the `Settings` class. E.g.

`from mitiq.calibration import ZNESettings
`
or 

`from mitiq.calibration import PECSettings`

based on the usual Python convention for names, it is hard to realize that they are not classes but just instances of the `Settings` class.

So, people may be tempted to do something like this:

`zne_settings = ZNESettings()
`
which is wrong. 

If you think this not an important problem, we can ignore and close this issue.

Otherwise, what about renaming the settings as `zne_settings`, `pec_settings` or `ZNE_SETTINGS`, `PEC_SETTINGS` ?

**Comment by Misty-W** [comment]
As constants, shouldn’t they be in all caps, i.e. `ZNE_SETTINGS`, `PEC_SETTINGS` ?

**Reply by natestemen** [reply]
Yes, technically I think this is "correct". I suggest one of us make this change next time we are modifying calibrator code.

**Comment by natestemen** [comment]
Completed in https://github.com/unitaryfund/mitiq/pull/2126/commits/5504756b0c5f4301b78ab09a424618875a975946.
